metrics/prometheus.go

```diff
@@ -1212,6 +1213,36 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 			}
 		}
 	}
+
+	if c.includedMetrics.Has(container.AppMetrics) {
+		for _, container := range containers {
+			cstats := container.Stats
+			if len(cstats) > 0 {
+				last := cstats[len(cstats)-1]
```

Comment:
I believe you want to use the first element in the list?  At least, that is what we do above...